README.md

```diff
@@ -515,36 +515,36 @@ Optional parameters to provide more metadata:
 - `-N Name`
 - `-C "Compositor description"`
 
-Arguments and metadata are stored in specifier unit drop-ins if needed.
```

Comment:
Here I wasn't sure what to change this to; it seems to work fine without this word.

Reply:
Specifier is a term from systemd documentation.